### PR TITLE
Add more info and example config to README

### PR DESCRIPTION
**Pull request by will-moore**
See https://trello.com/c/2bBx96kX/23-mapr-installation

Make the install docs more user-friendly with some background info and example how to set-up Map Annotations to work with a given config.

Staged at https://github.com/will-moore/omero-mapr/tree/README_more_info

TODO: I'll also look at using ```mapr``` to search for client-created Map Annotations as suggested elsewhere...

### CODE REVIEW

**Comment by will-moore** [comment]
After following workflow in README for image in Project/Dataset, I see

screen shot 2018-02-15 at 12 35 06

File: README.rst [hunk 1]
```diff
@@ -8,7 +8,22 @@
 MAPR
 ====
 
-An OMERO.web app allowing to browse the data through attributes linked to the image
+OMERO.map is an OMERO.web app that enables browsing of data through attributes linked to images
```
Comment (jburel) [inline README.rst]: mapr

File: README.rst [hunk 2]
```diff
@@ -8,7 +8,22 @@
 MAPR
 ====
 
-An OMERO.web app allowing to browse the data through attributes linked to the image
+OMERO.mapr is an OMERO.web app that enables browsing of data through attributes linked to images
+in the form of Map Annotations.
+
+It is used extensively by the `Image Data Resource <http://idr.openmicroscopy.org/>`_,
```
Comment (manics) [inline README.rst]: HTTPS instead? `https://idr`

**Comment by joshmoore** [comment]
The travis failure is due to a change in upstream pytest versions which I've fixed elsewhere.

**Comment by will-moore** [comment]
NB: I noticed that this user might benefit from using mapr. https://www.openmicroscopy.org/community/viewtopic.php?f=4&t=8446&p=19039
Can post a suggestion to look once this PR is merged.

**Comment by will-moore** [comment]
More screenshots to add for usage of "client" map annotations:

screen shot 2018-02-16 at 14 02 35
screen shot 2018-02-16 at 14 04 12

File: README.rst [hunk 3]
```diff
@@ -8,7 +8,25 @@
 MAPR
 ====
 
-An OMERO.web app allowing to browse the data through attributes linked to the image
+OMERO.mapr is an OMERO.web app that enables browsing of data through attributes linked to images
+in the form of Map Annotations.
+
+It is used extensively by the `Image Data Resource <https://idr.openmicroscopy.org/>`_,
+allowing users to find data by various categories such as Genes, Phenotypes, Organism etc.
```
Comment (mtbc) [inline README.rst]: odd that only the first two of the list are plural

**Comment by joshmoore** [comment]
Green! :tada:

**Pull request merged**